src/coins/eth.ts

```diff
@@ -0,0 +1,19 @@
+import {mnemonicToSeedSync} from 'bip39';
+import hdkey from 'hdkey';
+import {bufferToHex, privateToAddress} from 'ethereumjs-util';
+
+const HD_KEY_PATH = "m/44'/60'/3'/1/0";
+
+export const eth = {
+  keys: (passPhrase: string) => {
```

Comment:
It's one word. Use "passphrase". No need to camelCase. Check other places as well